pkg/relayer/proxy/server_builder.go

```diff
@@ -168,3 +169,23 @@ func (rp *relayerProxy) waitForSupplierToStake(
 
 	return supplier, nil
 }
+
+// logRelayMinerConfiguredServices logs the services configured in the RelayMiner
+// server configs. This is useful for debugging and understanding which services
+// the RelayMiner is configured to handle.
```

Comment:
// logRelayMinerConfiguredServices logs the services configured in the RelayMiner's configs.
// Useful for debugging and understanding which services the RelayMiner is configured to handle.